extensions/logitech-litra/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "logitech-litra",
+  "title": "Logitech Litra",
+  "description": "Manage Logitech Litra Glow and Logitech Litra Beam lights from Raycast",
+  "icon": "command-icon.png",
+  "author": "timrogers",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Manage Devices",
+      "description": "Turn your Logitech Litra devices on or off",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "cliDirectory",
+      "type": "textfield",
+      "required": true,
+      "title": "CLI directory",
```

Comment:
Hi @timrogers 👋 

Do you think it makes sense to make this not required?

Maybe set this a not required and only set it if needed; in the code, could we check for the default path (shouldn't it be the same almost always)

Reply:
I don't think it will be the same across machines - it depends on your Node installation. In my case, it's in `/Users/timrogers/.asdf/shims`, which I suspect is different to most machines!

Reply:
It'd be easiest if I could just check `PATH`, but I don't think that's going to work in Raycast given different shells people might be using with different config files.